### PR TITLE
HWS and SWS new directory

### PR DESCRIPTION
**Pull request by hamdanigbaria**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,64 +1,4 @@
-# mlx_steering_dump_parser
-This is Mellanox SW steering parser and triggering for dump files in **CSV format**.
-The supported dump files are those generated by ConnectX5 and ConnectX6DX.
+This is the HW/SW steering dump parsing tools.
```
Comment (kliteyn) [inline README.md:1]: need to rebase - readme file has some updates

**Review by kliteyn — CHANGES_REQUESTED** [review]
need to rebase

**Review by kliteyn — COMMENTED** [review]
looks good

File: hws/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+
+Required package
+===================
+For dumping the app HW resources:
+ -Python3
+ -Scapy
+ -Cython
+ -Pyverbs (Install RDMA-CORE after Cython)
+ -MFT (Higher than 4.20.0-00)
```
Comment (kliteyn) [inline hws/README.md]: (Higher than 4.20.0-00) - what does it mean?
4.20.0-00 is not good?
You mean "4.20.0 or higher"?

**Review by kliteyn — CHANGES_REQUESTED** [review]
please see my comment

File: hws/README.md [hunk 3]
```diff
@@ -61,5 +53,5 @@ Required package
  - Python3
  - Scapy
  - Cython
- - Pyverbs (Install RDMA-CORE after Cython)
+ - Pyverbs (Install RDMA-CORE after Cython, planned to be removed in August release)
```
Comment (kliteyn) [inline hws/README.md:56]: August release is now, so should it be here?

**Review by kliteyn — CHANGES_REQUESTED** [review]

File: hws/README.md (hunk 2 above)
Comment (hamdanigbaria) [reply hws/README.md]: updated in the next patches to 4.21

File: README.md (hunk 1 above)
Comment (hamdanigbaria) [reply README.md:1]: Dropped last patch that touch the SWS directories.
will merge this series then will move SWS files to another directory.

File: hws/README.md (hunk 3 above)
Comment (hamdanigbaria) [reply hws/README.md:56]: yes until it being released will update if any change happens.

File: README.md (hunk 1 above)
Comment (hamdanigbaria) [reply README.md:1]: done

**Review by kliteyn — APPROVED** [review]

**Pull request closed without merging**